src/common/types/store-data/assessment-card-selection-store-data.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { CardSelectionStoreData } from 'common/types/store-data/card-selection-store-data';
+
+export type AssessmentCardSelectionStoreData = CardSelectionStoreData;
```

Comment:
This should instead be an object with keys being assessment configuration keys and the properties being CardSelectionStoreData.